### PR TITLE
Update FJ video container [#180664396]

### PR DESCRIPTION
**Pull request by arcpixelpilot**
- Add `preload='metadata'` attribute to video
- Center video in containing div and make background black

### CODE REVIEW

**Comment by arcpixelpilot** [comment]
Once it's merged, I will create a front-end PR to update ckeditor5 package.

File: packages/ckeditor5-media-embed/src/mediaembedediting.js [hunk 1]
```diff
@@ -213,9 +213,12 @@ export default class MediaEmbedEditing extends Plugin {
 						const id = `fjvideo-${ match[ 2 ] }`;
 
 						return (
-							`<video id=${ id } style="max-width: 100%;" poster="${ posterUrl }" controls controlsList="nodownload">` +
-								`<source type="video/mp4" src=${ url }></source>` +
-							'</video>'
+							'<div style="position: relative; background-color: black; text-align: center;">' +
```
Comment (suvan92) [inline packages/ckeditor5-media-embed/src/mediaembedediting.js:216]: Why is the position relative?

**Review by suvan92 — COMMENTED** [review]
Also can we add screenshots please?

File: packages/ckeditor5-media-embed/src/mediaembedediting.js (hunk 1 above)
Comment (arcpixelpilot) [reply packages/ckeditor5-media-embed/src/mediaembedediting.js:216]: Not a big reason. Just use position relative as the other elements above.

**Review by suvan92 — APPROVED** [review]

**Pull request merged**